scripts/globals/items/beitetsu_parcel.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- ID: 6181
+-- Beitetsu Parcel
+-- Breaks up a Beitetsu Parcel
+-----------------------------------
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(4060, math.random(3, 15))
```

Comment:
Please don't use a magic number here, add an entry in `items.lua`, add the relevant require to the top of this file, and replace the magic number with `xi.items.BEITETSU`